src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -670,6 +671,10 @@ def setter(vm, external_identities=external_identities):
         vm.identity = VirtualMachineIdentity(type=identity_types)
         if external_identities:
             vm.identity.user_assigned_identities = {}
+            if not cmd.supported_api_version(min_api='2018-06-01', resource_type=ResourceType.MGMT_COMPUTE):
+                raise CLIInternalError("usage error: user assigned identity is not available under current profile.",
```

Comment:
A sentence should start with a capital letter.
Add a space between two sentences.